doc/source/user_guide/extensions.rst

```diff
@@ -0,0 +1,62 @@
+.. _ref_user_guide_extension:
+
+Extensions
+==========
+
+This page contains the extensions shipped extensions that seamlessly integrate with this theme to enhance 
+documentation functionality 
+
+Linkcode extension
+-------------------
+
+The Linkcode extension automatically adds "View Source" links to the documentation for Python, C, C++, 
+and JavaScript objects. It allows you to link to the source code hosted on GitHub.
+
+Installation
+~~~~~~~~~~~~
+
+To use the ``ansys-sphinx-theme`` linkcode extension, you need to add it to the Sphinx configuration:
+
+#. Add the following line in the ``conf.py`` file:
+   .. code-block:: python
+
+      extensions = ["ansys_sphinx_theme.extension.linkcode"]
+
+Configuration options
+---------------------
+
```

Comment:
You should also explain the "priorities" for both config options.

If both configs are given, which one takes precedence?

Direct and explicit config should take precedence: i.e. ``link_code_library`` et al.
Clarify this in the docs please.

Reply:
Thats true, i added. Thanks @RobPasMue